tutor/templates/jobs/init/lms.sh

```diff
@@ -1,5 +1,16 @@
+# If the the entry points info file (within the *egg-info directory)
+# doesn't exist or is out-of-date with respect to setup.py,
+# then we need to (re-)generate the egg-info by pip-installing this directory.
+# Otherwise, XBlocks and Django App Plugins will not be registered.
+# (This will only ever apply when a local copy of edx-platform has been
+#  bind-mounted by the user.)
+ENTRY_POINTS_INFO=Open_edX.egg-info/entry_points.txt
+if [ ! -f "$ENTRY_POINTS_INFO" ] || [ "$ENTRY_POINTS_INFO" -ot setup.py ]; then
+  pip install -e .
+fi
```

Comment:
Note:

I took this block directly from the other PR: https://github.com/kdmccormick/tutor/pull/24/file

In the context of `do init`, here's no harm in re-running `pip install -e .`, other than the ~2 seconds it take to run, so we could simplify this block to just:
```suggestion
pip install -e .
```